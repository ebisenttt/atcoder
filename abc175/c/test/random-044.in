568699808676406 721831784286633 179592306151297
