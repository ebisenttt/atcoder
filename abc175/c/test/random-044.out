29922890222515
